sim/paladin/paladin.go

```diff
@@ -77,18 +81,27 @@ func (paladin *Paladin) Initialize() {
 	paladin.registerHammerOfWrath()
 	paladin.registerHolyWrath()
 	paladin.registerAvengingWrath()
+	paladin.registerAuraMastery()
+
+        if paladin.primaryPaladinAura == proto.PaladinAura_SanctityAura {
+            paladin.currentPaladinAura = core.SanctityAuraAura(paladin.GetCharacter())
+        }
```

Comment:
Looks like the indentation is a bit off here too

Reply:
Also I think we should go ahead and do this inside the `Reset()` similar to how seals are handled. That way if there's ever a reason to add the ability to switch seals it can be reset between each iteration

Reply:
I can add that